samples/Azure.Management.Storage/Generated/ImmutabilityPolicy.cs

```diff
@@ -92,7 +92,7 @@ public virtual ImmutabilityPolicyData Data
         /// <summary> Gets the existing immutability policy along with the corresponding ETag in response headers and body. </summary>
         /// <param name="ifMatch"> The entity state (ETag) version of the immutability policy to update. A value of &quot;*&quot; can be used to apply the operation only if the immutability policy already exists. If omitted, this operation will always be applied. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public async virtual Task<Response<ImmutabilityPolicy>> GetAsync(string ifMatch = null, CancellationToken cancellationToken = default)
+        public async virtual Task<Response<ImmutabilityPolicy>> GetAsync(ETag? ifMatch = null, CancellationToken cancellationToken = default)
```

Comment:
Would we change the method signatures of existing GA libraries in management plane?  This would be a breaking change.